app/components/jobseekers/schools_overview_component.html.haml

```diff
@@ -0,0 +1,88 @@
+%section#school-overview.govuk-tabs__panel.no-border-override
+  %h3.govuk-heading-l
+    = t('schools.schools_overview')
+
+  %table.govuk-table
+    %tbody.govuk-table__body
+      %tr.govuk-table__row
+        %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
+          = t('school_groups.type')
+        %td.govuk-table__cell
+          = @vacancy.parent_organisation.group_type
+      - if @vacancy.contact_email.present?
+        %tr.govuk-table__row
+          %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
+            = t('jobs.contact_email')
+          %td.govuk-table__cell
+            = mail_to @vacancy.contact_email, @vacancy.contact_email, class: 'govuk-link link-wrap', subject: t('jobs.contact_email_subject', job: @vacancy.job_title), body: t('jobs.contact_email_body', url: job_url(@vacancy))
+
+  - if @vacancy.about_school.present?
+    %h4.govuk-heading-s
+      = t('schools.about_schools')
+    %p= vacancy_or_organisation_description(@vacancy)
+
+  %h4.govuk-heading-s
+    = t('schools.school_details')
+
+  #accordion-default.govuk-accordion{"data-module" => "govuk-accordion"}
+    - @vacancy.organisations.each_with_index do |organisation, index|
+      .govuk-accordion__section
+        .govuk-accordion__section-header
+          %h3.govuk-accordion__section-heading
+            %span.govuk-accordion__section-button{ id: "accordion-default-heading-#{index}"}
+              = organisation.name
+        .govuk-accordion__section-content{ id: "accordion-default-content-#{index}", aria: { labelledby: "accordion-default-heading-1" } }
```

Comment:
i think this should be `"accordion-default-heading-#{index}` so it matches the id above